mule-packager/src/main/java/org/mule/tools/api/validation/ProjectValidatorFactory.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Mule ESB Maven Tools
+ * <p>
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * <p>
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.tools.api.validation;
+
+import org.mule.tools.api.packager.ProjectInformation;
+
+import org.mule.maven.client.internal.AetherMavenClient;
+import org.mule.tools.api.classloader.model.SharedLibraryDependency;
+import org.mule.tools.api.packager.packaging.PackagingType;
+import org.mule.tools.api.util.Project;
+
+import java.util.List;
+
+import static org.mule.tools.api.packager.packaging.PackagingType.MULE_DOMAIN_BUNDLE;
+
+public class ProjectValidatorFactory {
```

Comment:
Please add a unit test for this class 
the idea here is to validate that baed on the projectInformation the returned object is of the correspondent type